openstack/clustering/v1/webhooks/requests.go

```diff
@@ -0,0 +1,43 @@
+package webhooks
+
+import (
+	"github.com/gophercloud/gophercloud"
+)
+
+// TriggerOpts Webhooks request parameters
+type TriggerOpts struct {
+	// V (Optional)	query	string	The webhook implementation version requested.
+	V string `q:"V, omitempty"`
```

Comment:
While the API documentation shows `V` as a valid query parameter, I don't see it defined in the actual Python code:

https://github.com/openstack/senlin/blob/e5bc16b3fea2f10dead365f983c5a937f23d72b0/senlin/api/openstack/v1/webhooks.py#L28

Do you know if V might have been an old parameter?

Reply:
Contacted senlin forum.